src/Compiler/FSComp.txt

```diff
@@ -1462,7 +1462,7 @@ keywordDescriptionStatic,"Used to indicate a method or property that can be call
 keywordDescriptionStruct,"Used to declare a structure type. Also used in generic parameter constraints."
 keywordDescriptionThen,"Used in conditional expressions. Also used to perform side effects after object construction."
 keywordDescriptionTo,"Used in for loops to indicate a range."
-keywordDescriptionTry,"Used to introduce a block of code that might generate an exception. Used together with with or finally."
+keywordDescriptionTry,"Used to introduce a block of code that might generate an exception. Used together with "with" or "finally"."
```

Comment:
Maybe better single quotes, 'with' or 'finally', to be on the safe side.

Reply:
Good call. Looking more closely all the other instances in this file used `\"`. And I wasn't remotely attached to the specific quotation marks. -- The file has *way* more instances of `'`.

Anyway, changed locally. (I've also rebased locally to deal w/ a single conflict.)

I'm holding off pushing for now...